01_fib/pokkalar/main_test.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+)
+
+func TestMain(t *testing.T) {
+	var b bytes.Buffer
+	out = &b
+	expected := "1\n1\n2\n3\n5\n8\n13\n"
```

Comment:
Using raw strings may be better for readability. Like Julia mentioned.
Eg:
`1
2
3
`

Reply:
Done.